oragono.yaml

```diff
@@ -144,53 +144,42 @@ server:
         # this works around that bug, allowing them to use SASL.
         send-unprefixed-sasl: true
 
-    # maximum number of connections per subnet
-    connection-limits:
-        # whether to enforce connection limits or not
-        enabled: true
+    # IP-based DoS protection
+    ip-limits:
+        # whether to enforce limits on the total number of concurrent connections per IP / CIDR
+        limit: true
```

Comment:
having the section named `ip-limits:` and this key named `limit:` feels redundant and a bit confusing. maybe we could name this whole section `dos-protection` and this key `concurrent-limits` or something? I dunno, this is a hard key to name I think

Reply:
What if we just called the boolean switch `count` instead of `limit`? Like `count: true`.

Reply:
that cooould work, so long as the comment was changed in kind. maybe "whether to count and limit the number of simultaneous connections" sounds alright?

Reply:
"whether to limit the total count of concurrent connections per IP/CIDR"?

Reply:
let's go with "whether to limit the total number of concurrent connections per IP/CIDR", that explains what the key does as clearly as it can be